bin/create-dotenv

```diff
@@ -1,27 +1,51 @@
 #!/bin/sh
 
 if [ -f .env ]; then
-  echo "Skipping generating .env file because it already exists."
+    echo "Skipping generating .env file because it already exists."
 else
-  cat <<EOF > .env
+    cat <<EOF > .env
 # Random string used to sign cookies.
 COOKIE_SECRET=$(hexdump -n 128 -e '"%08x"' /dev/random)
 
 # Random string used for encrypting and decrypting Earthdata login tokens
 # sent to backend services
 SHARED_SECRET_KEY=$(hexdump -n 16 -e '"%08x"' /dev/random)
 
-# See the "OAuth 2 (Earthdata Login)" section in the env-defaults file
-# Contact a harmony developer if unsure what values to use
-OAUTH_CLIENT_ID=
-OAUTH_UID=
-OAUTH_PASSWORD=
 EOF
+    cat <<PROMPT_EDL
```

Comment:
Nit: spacing is pretty wonky in this section of code, and, I guess, above, with everything in an if-statement and then HEREDOCs.  I _think_ you can use the `<<-` syntax for HEREDOCs to trim leading whitespace, so the only unindented thing is the final `EOF` of the block

Reply:
(I realize this code is all copy-paste)

Reply:
fixed